fgd/point/comp/comp_vactube_beizer.fgd

```diff
@@ -0,0 +1,48 @@
+@MoveClass base(StaticTargetname, Angles)
+	sphere(radius)
+	animator() keyframe()
+	iconsprite("editor/comp_prop_rope")
+	appliesto(srctools)
+= comp_vactube_bezier: "Constructs a custom vactube model, using a set of path points as bezier keyframes. Config values are only read from the starting point"
```

Comment:
Since there aren't any keyvalues in `comp_vactube_bezier` that are unique to it, it might be simpler to just add an additional interpolation type to that field, avoid the extra entity that people need to remember.

Reply:
I agree. I used this method to get this working quickly without looking into how config works but now that I want to implement separate glass I should make this a config instead of entity.

Reply:
In regards to that, note that the first parameter to `get_model()` (the "key") is stored along with the model, to allow them to be reused. So you'd need to pass it in there.